AlgorithmAnalyzer/Backend/main.py

```diff
@@ -43,56 +51,102 @@ def list_routes():
         return list_routes()
 
 
-@app.route("/audio/train", methods=['GET', 'POST'])
-def handling_audio_train_endpoint():
-    """ This handles generic operations that have to do with audio
+@app.route("/users", methods=['GET'])
+def handle_users_endpoint():
+    """
+    serve list of registered users
+    """
+    return {'users': sample_manager.get_all_usernames()}, status.HTTP_200_OK
+
+
+@app.route("/audio/<type>", methods=['POST'])
```

Comment:
```suggestion
@app.route("/audio/<string:type>", methods=['POST'])
```
Może z walidacją typu tutaj? Zwróci samo 404

Reply:
`'/audio/<type>' === ' /audio/<string:type>'`  [doc](http://flask.pocoo.org/docs/1.0/api/#url-route-registrations)

Reply:
W sumie tak :D. Ale może więcej informacji dla przeglądania API? Widać w adresie flaskAPI wtedy

Reply:
uuuu, racja, zmienione